cloud/workspace/workspace_token.go

```diff
@@ -248,7 +261,7 @@ func RotateToken(name, workspace string, force bool, out io.Writer, client astro
 	}
 	fmt.Fprintf(out, "Astro Workspace Token %s was successfully rotated\n\n", token.Name)
 	ApiToken := resp.JSON200
-	fmt.Println(ApiToken.Token)
+	fmt.Println(*ApiToken.Token)
```

Comment:
Instead of printing the token which would be always available in cicd logs, can we directly set the `ASTRO_API_TOKEN` and eliminate another user action?